splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/FromFinalTableIT.java

```diff
@@ -122,9 +127,16 @@ public static void createData(Connection conn, String schemaName) throws Excepti
 
     @Before
     public void createDataSet() throws Exception {
+        runningOperations = getNumberOfRunningOperations();
```

Comment:
Shouldn't this always be 0, since the test is Serial?

Reply:
Theoretically.  But the user may be doing something else on the machine.  If they have another running operation in a user session I don't want it to raise a false alarm in this test.